snap/local/snap-runner

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+
+set -- --middleware "$(snapctl get middleware)" "$@"
+set -- --verbose "$(snapctl get verbosity)" "$@"
+
+system="$(snapctl get system)"
```

Comment:
Change `system` occurrences to `transport` (see comment below). Apply the same criteria to subsequent occurrences.

Reply:
Easy peasy: done.